robosuite/scripts/tune_camera.py

```diff
@@ -0,0 +1,259 @@
+        """
+        Store internal state here.
+        """
+        self.env = env
+        self.camera_id = camera_id
+
+    def on_press(self, window, key, scancode, action, mods):
+        """
+        Key handler for key presses.
+        """
+
+        # controls for moving position
+        if key == glfw.KEY_W:
+            # move forward
+            move_camera(env=self.env, direction=[0., 0., -1.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_S:
+            # move backward
+            move_camera(env=self.env, direction=[0., 0., 1.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_A:
+            # move left
+            move_camera(env=self.env, direction=[-1., 0., 0.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_D:
+            # move right
+            move_camera(env=self.env, direction=[1., 0., 0.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_R:
+            # move up
+            move_camera(env=self.env, direction=[0., 1., 0.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_F:
+            # move down
+            move_camera(env=self.env, direction=[0., -1., 0.], scale=DELTA_POS_KEY_PRESS, camera_id=self.camera_id)
+
+
+        # controls for moving rotation
+        elif key == glfw.KEY_UP:
+            # rotate up
+            rotate_camera(env=self.env, direction=[1., 0., 0.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_DOWN:
+            # rotate down
+            rotate_camera(env=self.env, direction=[-1., 0., 0.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_LEFT:
+            # rotate left
+            rotate_camera(env=self.env, direction=[0., 1., 0.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_RIGHT:
+            # rotate right
+            rotate_camera(env=self.env, direction=[0., -1., 0.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_PERIOD:
+            # rotate counterclockwise
+            rotate_camera(env=self.env, direction=[0., 0., 1.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+        elif key == glfw.KEY_SLASH:
+            # rotate clockwise
+            rotate_camera(env=self.env, direction=[0., 0., -1.], angle=DELTA_ROT_KEY_PRESS, camera_id=self.camera_id)
+
+
+    def on_release(self, window, key, scancode, action, mods):
+        """
+        Key handler for key releases.
+        """
+        pass
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--env",
+        type=str,
+        default="SawyerLift",
+    )
+    args = parser.parse_args()
+
+    # read camera XML tag from user input
+    inp = input("Please paste the current camera xml tag below (e.g. <camera ... />):\n")
```

Comment:
This script is very functional but not well documented. Consider the following two options (or both):

1. Provide a concrete example, e.g., go to a MJCF file in /models/assets/arenas and copy a camera tag
2. Allow blank input, which uses a default camera initial pose (e.g., the frontview)